### PR TITLE
Update docs 169

### PR DESCRIPTION
**Pull request by jimzucker**
#169 Updated Readme for using Docker containers.

### CODE REVIEW

**Comment by jimzucker** [comment]
Rebased to clear up conflicts with master.

File: jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java [hunk 1]
```diff
@@ -21,18 +21,10 @@
...
 import java.util.Set;
 
-import static org.mockito.Matchers.anyListOf;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.verifyZeroInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (jdamick) [inline jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java]: we dont use *, please list them out

File: jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java [hunk 2]
```diff
@@ -89,46 +81,6 @@ public void collect_oneJob_exists_notAdded() {
         verify(hudsonJobRepository, never()).save(job);
     }
 
-
-    @Test
-    public void delete_job() {
```
Comment (jdamick) [inline jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java]: why are these removed?

File: jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java [hunk 3]
```diff
@@ -89,46 +81,6 @@ public void collect_oneJob_exists_notAdded() {
...
-        when(hudsonJobRepository.findByCollectorIdIn(udId)).thenReturn(jobs);
-        when(dbComponentRepository.findAll()).thenReturn(components());
-        task.collect(collector);
-        List<HudsonJob> delete = new ArrayList<>();
-        delete.add(job2);
-        verify(hudsonJobRepository, times(1)).delete(delete);
-    }
-
-    @Test
-    public void delete_never_job() {
```
Comment (jdamick) [inline jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java]: why removed?

File: jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/DefaultHudsonClientTests.java [hunk 4]
```diff
@@ -75,22 +75,6 @@ public void joinURLsTest() throws Exception {
     }
 
     @Test
-    public void rebuildURLTest() throws Exception {
```
Comment (jdamick) [inline jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/DefaultHudsonClientTests.java]: why is this removed?

**Comment by jdamick** [comment]
this needs to be rebased..

**Comment by jimzucker** [comment]
I rebased again, not sure what I did wrong earlier today.  Please check after CI finishes, have a great weekend.

**Comment by jimzucker** [comment]
I see the problem if you look at commit :
| *   b497082 - Merge remote-tracking branch 'upstream/master' (6 days ago) amp5208

screenshot 2016-02-07 09 18 09


It does not appear to have been rebased correctly and it deleted previous changes:
screenshot 2016-02-07 09 17 25

**Comment by jimzucker** [comment]
I am going to close this pull request as the master has lost some changes. I will consolidate and test everything in one branch.
